### DISCUSSION TITLE
Welcome to Open-Llama Discussions!

### DISCUSSION THREAD

**Discussion by s-JoL** [s-JoL/Open-Llama#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by YaBoyJ** [comment]
Hey, do you know if anyone has setup a mirror to the wudao dataset? I don't have a WeChat account and so can't sign in. 

s-Jol, thanks for all the work you've put into this!

**Reply by s-JoL** [reply]
First of all, thank you for your interest in our project. After searching, I did not find a complete mirror of the Wudao dataset. There is a mirror on HuggingFace that only contains part of the data: https://huggingface.co/datasets/suolyer/wudao. Also, I have read the relevant agreement of the Wudao dataset, which restricts its use outside of China: https://data.baai.ac.cn/resources/agreement/BAAIDataAgreement.pdf.

Finally, the Wudao dataset mainly contains Chinese data. If Chinese is not necessary for your project, I suggest using only The Pile or RedPajama: https://huggingface.co/datasets/togethercomputer/RedPajama-Data-1T to do pretraining.

**Reply by s-JoL** [reply]
If you do need Chinese data but are unable to obtain the Wudao dataset, you may consider using this one: https://github.com/CLUEbenchmark/CLUECorpus2020.

**Reply by YaBoyJ** [reply]
Awesome, thank you so much, I'll take a look at that dataset!

**Comment by ZacharyDK** [comment]
Development branch (4.29) of transformers is required to run s-JoL/Open-Llama-V1 . This isn't mentioned on the hugging face page. Trying to run s-JoL/Open-Llama-V1 gives a open llama key error.  

Had to find this information on github. After upgrading to the development branch of the transformers module, the large bin files are downloading. 

***

BTW thanks for doing the legwork for getting a variant of open llama onto hugging face + InstructGPT back end. This makes the model widely available.

**Reply by s-JoL** [reply]
![image](https://user-images.githubusercontent.com/16948304/236654242-7dad6d94-19ca-4824-a472-e00859d04d59.png)
Thank you for your interest in our project. In fact, there is a small text message at the top of the Hugging Face page that reminds users to install transformers from source. However, we agree that the message may not be prominent enough. We will consider how to give a more visible prompt to users.

**Comment by ycat3** [comment]
Hi, it's great work.
Open-Llama-V2 is working on my desktop PC (ubuntu22.04).
Llama-V2 speaks Japanese  well.
Is Open-Llama-V2-pretrain model fine-tuned model for Chinese language ?
I am a contributor for [lm-sys/FastChat](https://github.com/lm-sys/FastChat)(Vicuna).
Vicuna is also based on llama.
Vicuna-13b is pretty better than Vicuna-7b.
Will you plan to release 13b model as well ?

**Reply by s-JoL** [reply]
Yes, English and Chinese data are mainly used in finetune. I have tried to increase the ability of Japanese and Korean through the xp3 data set, but I have not succeeded in finding ja and ko. 

I'm still thinking about the larger model, because it's not clear how big the model is enough, and whether using peft on the larger model will be better than using full finetune on the small model. Do you have some relevant experience in fastchat, what advice do you have for this?

**Comment by ycat3** [comment]
FastChat/Vicuna releases 7b and 13b model and it seems no more actual schedule to release 30b.
Vicuna-13b requires at least 28Gib GPU VRAM to inference.
My main concern is enable to inference vicuna-13b by consumer-class GPUs, such as  multiple rtx3060. 
It's successful now because my PR is merged to main tree.
Vicuna can be trained on 8 A100 GPUs with 80GB memory.
As usual GPU resources are limited for open source community.
There is tradeoff.
